docs/CHANGELOG.md

```diff
@@ -1,11 +1,23 @@
 ## Next
 
+* New Rich Texts added for Lost Sectors and Stasis
+* Vendor fixes (misc)
```

Comment:
Show reasons why you can't buy vendor items, and grey out bounties that you've already purchased on the vendors screen.